src/components/createCredentialDialog/createCredentialDialog.js

```diff
@@ -226,6 +226,7 @@ const CreateCredentialDialog = ({
       {values[apiTypes.API_QUERY_TYPES.CREDENTIAL_TYPE] === 'network' && (
         <FormGroup label={t('form-dialog.label', { context: ['auth-type', 'create-credential'] })}>
           <DropdownSelect
+            ouiaId="auth_type"
```

Comment:
looks good!

optional: 
In the future consider modifying the `DropdownSelect` component to handle or apply the `ouiaId` when an `id` or `name` property is applied to `DropdownSelect`. It appears that functionality is missing currently.

The reason this thought popped up is that the `ouiaId` appears to be the same parameter name used within the form state